utils/OptionUtils.js

```diff
@@ -50,6 +53,20 @@ function areRequiredOptionsSet(opts) {
     });
 }
 
+function ensureReportersConfig(opts) {
+    // Only set the default reporter when no explicit reporter configuration is provided
+    if(!opts.hasOwnProperty('reporters')) {
+        opts.reporters = DEFAULT_REPORTER;
+    }
+}
+
+function ensureIgnoreConfig(opts) {
+    // Ignore the strict mode keyword mutations, unless the mutateStrictModeKeyword option has been provided
+    if(!opts.mutateStrictModeKeyword) {
```

Comment:
What if we want to add another string to the DEFAULT_IGNORE? That will then also be overridden with 'mutateStrictModeKeyword' option.

This option is very specific. Wouldn't it be better to replace it with a more general option - something like 'discardDefaultIgnore'?